src/sentry/release_health/tasks.py

```diff
@@ -85,19 +85,18 @@ def adopt_releases(org_id: int, totals: Totals) -> Sequence[int]:
     ):
         for project_id, project_totals in totals.items():
             for environment, environment_totals in project_totals.items():
-                total_releases = len(environment_totals["releases"])
                 for release_version in environment_totals["releases"]:
                     # Ignore versions that were saved with an empty string
                     if not Release.is_valid_version(release_version):
                         continue
 
-                    threshold = 0.1 / total_releases
                     if (
                         environment
                         and environment_totals["total_sessions"] != 0
-                        and environment_totals["releases"][release_version]
-                        / environment_totals["total_sessions"]
-                        >= threshold
+                        and has_been_adopted(
+                            environment_totals["total_sessions"],
+                            environment_totals["releases"][release_version],
+                        )
```

Comment:
👍 to using function calls to describe the logic. one small step for readable code, and a giant leap in bug fixing.